Resources/Prototypes/Entities/Mobs/Species/shadowkin.yml

```diff
@@ -227,16 +228,17 @@
     - type: Shadowkin
     - type: Psionic
       mindbreakingFeedback: shadowkin-blackeye
-    - type: InnatePsionicPowers
-      powersToAdd:
-        - DarkSwapPower
     - type: LanguageKnowledge
       speaks:
         - TauCetiBasic
         - Marish
+        - Empathy
```

Comment:
```suggestion
```
I would actually overwhelmingly prefer that Empathy chat be outright deleted from the game and replaced with Shadowkin just having roundstart Telepathy. It's mind numbingly awful that there's literally two telepath chats.